core/src/main/kotlin/org/evomaster/core/problem/external/service/ExternalServiceInfo.kt

```diff
@@ -14,6 +19,10 @@ class ExternalServiceInfo(
         }
     }
 
+    fun assignWireMockServer(wm: WireMockServer) {
+        wireMockServer = wm
```

Comment:
why not done in constructor? leaving potentially uninitialized `lateinit` fields is risky